src/main/java/ch/njol/skript/expressions/ExprProtocolVersion.java

```diff
@@ -59,7 +59,7 @@
 public class ExprProtocolVersion extends SimpleExpression<Long> {
 
 	static {
-		Skript.registerExpression(ExprProtocolVersion.class, Long.class, ExpressionType.SIMPLE, "[the] [(sent|required|fake)] protocol version [number]");
+		Skript.registerExpression(ExprProtocolVersion.class, Long.class, ExpressionType.SIMPLE, "[the] [server] list [(sent|required|fake)] protocol version [number]");
```

Comment:
Is this done to make the difference between the two expressions more clear? If so, you shouldn't make it a breaking change, maybe just the `[server]` part is good enough?

Reply:
Wouldn't that cause a syntax conflict between ExprPlayerProtocolVersion and ExprProtocolVersion?

Reply:
No, ExprPlayerProtocolVersion has the syntax `of %players%` / `%players%'[s]` added, which ExprProtocolVersion doesn't have